algosdk/v2client/algod.py

```diff
@@ -641,6 +646,78 @@ def simulate_raw_transactions(
         )
         return self.simulate_transactions(request, **kwargs)
 
+    def get_sync_round(self, **kwargs: Any) -> AlgodResponseType:
+        """
+        Get the minimum sync round for the ledger.
+
+        Returns:
+            Dict[str, Any]: Response from algod
+        """
+        req = "/ledger/sync"
+        return self.algod_request("GET", req, **kwargs)
+
+    def set_sync_round(self, round: int, **kwargs: Any) -> AlgodResponseType:
+        """
+        Set the minimum sync round for the ledger.
+
+        Args:
+            round (int): Sync round
+
+        Returns:
+            Dict[str, Any]: Response from algod
+        """
+        req = f"/ledger/sync/{round}"
+        return self.algod_request("POST", req, **kwargs)
+
+    def unset_sync_round(self, **kwargs: Any) -> AlgodResponseType:
+        """
+        Unset the minimum sync round for the ledger.
+
+        Returns:
+            Dict[str, Any]: Response from algod
+        """
+        req = "/ledger/sync"
+        return self.algod_request("DELETE", req, **kwargs)
```

Comment:
DELETE does not work on Python right now - might be because `http.server` does not have a do_delete() method: https://docs.python.org/3/library/http.server.html#http.server.SimpleHTTPRequestHandler.do_GET

Reply:
resolved - add do_delete method in the mock server